src/mjan.c

```diff
@@ -305,6 +305,20 @@ USERINPUT GetAndPrintPlayersMove(POSITION position, MOVE *move, STRING playerNam
  * @return TRUE iff the input is a valid text input.
  */
 BOOLEAN ValidTextInput(STRING input) {
+    int start_pos, end_pos;
+
+    if (sscanf(input, "%d %d", &start_pos, &end_pos) != 2) {
```

Comment:
https://cplusplus.com/reference/cstdio/sscanf/
"This count can match the expected number of items or be less (even zero) in the case of a matching failure."